### PR TITLE
Add more options for configuring single file report

### LINKED ISSUE #17: Wrong file path format
I really like this gem in combination with aircover.co, but your generated path is invalid because the LCOV specification says you have to use absolute paths. You are overwriting this with ./ at https://github.com/fortissimo1997/simplecov-lcov/blob/master/lib/simplecov-lcov.rb#L72

How can we fix that?

### PR DESCRIPTION
**Pull request by povilasjurcys**
This improvement allows to customize single file path. It should fix #17. It's also related (kind of duplicated) with #19 #18 and #16, since it allows to do all the things mentioned in other PR's

### CODE REVIEW

File: lib/simplecov-lcov.rb [hunk 1]
```diff
@@ -12,53 +13,59 @@ class LcovFormatter
...
         end
 
-        # Output path for single file report.
-        # ==== Return
-        # Path for output path of single file report.
-        def single_report_path
-          basename = Pathname.new(SimpleCov.root).basename.to_s
-          File.join(output_directory, "#{basename}.lcov")
+        def report_with_single_file=(value)
+          config.report_with_single_file = value
```
Comment (povilasjurcys) [inline lib/simplecov-lcov.rb]: This method is left only to support older versions. I would like to deprecate it. @fortissimo1997, what is your opinion on this?

Comment (fortissimo1997) [reply lib/simplecov-lcov.rb]: @povilasjurcys 
This method needs to be deprecated, I think.
If so, next version has to be 1.0.0!

Comment (povilasjurcys) [reply lib/simplecov-lcov.rb]: I added deprecation warning for now

**Comment by fortissimo1997** [comment]
👍

**Pull request merged**